aws/sdk/examples/medialive/src/bin/helloworld.rs

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+/// Lists your AWS Elemental MediaLive input names and ARNs.
```

Comment:
I think this needs to be prefixed with `media-live`? I was hitting issues where you could only have one binary with a name in a workspace